scripts/zones/Bibiki_Bay/mobs/Dalham.lua

```diff
@@ -7,13 +7,44 @@ local entity = {}
 
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180)
+end
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:setMod(xi.mod.SLEEPRES, 90)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.SILENCE)
+    mob:addImmunity(xi.immunity.TERROR)
     mob:setMod(xi.mod.WATER_ABSORB, 100)
 end
 
+entity.onAdditionalEffect = function(mob, target, damage)
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.ENWATER, { chance = 100 })
+end
+
 entity.onMobFight = function(mob, target)
-    -- "Increases swings per attack round as its HP lowers. At 15-20% swings like Charybdis giving the illusion of Hundred Fists."
-    local swings = 1 + math.floor((100 - mob:getHPP()) / 18)
-    mob:setMobMod(xi.mobMod.MULTI_HIT, swings)
+    if mob:getHPP() < 75 and mob:getLocalVar('multihitControl') == 0 then
+        mob:setLocalVar('multihitControl', 1)
+        -- set DOUBLE_ATTACK to 100
+        -- cannot use MULTI_HIT because that is just upto X hits
+        mob:setMod(xi.mod.DOUBLE_ATTACK, 100)
+        -- dummy mob ability for 2hr animation (does not use tp)
+        mob:useMobAbility(626)
```

Comment:
Based on the cap there's no way this is actually a real ability and you should just use injectActionPacket

`mob:injectActionPacket(mob:getID(), 11, 437, 0, 24, 0, 0x18, 626)`
most 2hr dustclouds like this should probably just be injections

(note the back to back WS/finish packets)
![image](https://github.com/user-attachments/assets/c55bda13-9bf2-42cb-8515-16da9728f119)

Reply:
> Based on the cap there's no way this is actually a real ability and you should just use injectActionPacket
> 
> `mob:injectActionPacket(mob:getID(), 11, 437, 0, 0x18, 0, 0, 626)` most 2hr dustclouds like this should probably just be injections

Thanks for the info, I have updated the PR to use `injectActionPacket` instead!